tasks/jenkins.js

```diff
@@ -21,10 +21,15 @@ module.exports = function (grunt) {
     process.env.PATH = path.join(delimiter);
   });
 
+  grunt.registerTask('jenkins:docs', [
+    'docker:docs'
+  ]);
+
   grunt.registerTask('jenkins:unit', [
     'jenkins:env',
     'rejectRejFiles',
 
+    'docker:docs',
```

Comment:
I'll remove this before merging - I'm leaving it for now to make sure CI goes green until we have a docs pipeline.